EIPS/eip-erps/eip-erps.md

```diff
@@ -0,0 +1,194 @@
+
+#### weiTransfer
+Transfers eth from one address to another.
+
+*Data fields*
+
+- **type** (*string*): `weiTransfer`
+- **fromAddress** (*hex string*): The address from which each should be transferred
+- **toAddress** (*hex string*): The address to which eth should be sent
+- **value** (*decimal string*): The amount to be transferred, in units of wei. The value must be a whole number and non-negative.
+
+
+#### storeCode
+Stores the given code at the given address.
+
+*Data fields*
+
+- **type** (*string*): `storeCode`
+- **toAddress** (*hex string*): The address to which the contract should be restored.  
+- **expectedCodeHash** (*hex string*): the expected hash of the code already at the toAddress.The empty string should be used if no code is expected at the toAddress.  In all other cases, the “0x” prefix is optional.
+- **code** (*hex string*): the new byte code associated with the contract
+
+### Appendix (Optional)
+The appendix can include additional supporting evidence or attachments that will help reviewers understand or verify the claims made in the ERP.  
+
+For the storeCode operation, it should include the proposed contract source (e.g. solidity) as well as the other details required such that a reviewer can compile the source and generate the same byte code.  It should also include the source that was originally stored at that address, if possible/applicable.  If the two contracts are not identical, changes should be noted. If they are identical, the author should indicate why no changes are necessary (this is unlikely). Additionally, any relevant reviews, audits, and test cases should be included to the extent that they address the issues encountered with the original contract.  
+
+If accepted, an ERP could easily compile the block at while the changes are to take place, and the source of the actions (which would be the output of the script, in a standardized object output). These can be bundled with the client for seamless execution.
+
+
+## ERP Approval Process
+Approval of any ERP will require a mandatory minimum of 30-days between the time the ERP is submitted (or substantially modified) and adoption by clients, and ERPs affecting more funds or more addresses may prompt a considerably longer review period.  In practice, ERPs will often require more time because the will likely be bundled into other client upgrades that are on their own schedule.  However, the 30-day mandatory review period  guarantees a period for comment and auditing and prevents any last minute inclusions from being rushed into an upcoming upgrade.  
+
+Once an ERP is published (via a pull request made to the EIP repository), the approval process begins.  The author is encouraged to seek feedback from the following groups:
```

Comment:
[comment not as an EIP editor] Why does this EIP specify these two groups?  Why shouldn't all Ethereum users be heard?

Reply:
This guideline states the minimum requirements that ERP authors must meet.  Other people can (and should!) contribute to the discussion with substantive feedback relevant to the specific claims.  i.e. to verify or dispute the facts presented with supporting evidence.  Non-substantive feedback (e.g. "these guys are idiots" or some emoji) are of little value and can be easily manipulated.

Cases that fall under the ERP process are specifically ones that are factual in nature and can be handled through a process more closely resembling peer-review than a voting process.  Recovery cases that are not factual in nature (i.e. where ownership cannot be proved) are intentionally not covered.